web-admin/src/features/public-urls/CreatePublicURLForm.svelte

```diff
@@ -86,6 +97,18 @@
             `${window.location.origin}/${organization}/${project}/-/share/${token}`,
             "URL copied to clipboard",
           );
+
+          void queryClient.refetchQueries(
+            getAdminServiceListMagicAuthTokensQueryKey(organization, project),
+          );
+
+          eventBus.emit("notification", {
+            message: "Public URL created",
+            link: {
+              href: `/${organization}/${project}/-/settings/public-urls`,
+              text: "Go to public URLs",
+            },
+          });
```

Comment:
UX feedback: I'd prefer we didn't show this notification. After I submit the "Create and copy URL" button, my next desired action is either 1) to open a new tab and paste in the URL to double-check it, or 2) paste the URL into a messaging app. Seeing this prompt to navigate to the management page makes me feel conflicted.